dev-tools/mage/dockerbuilder.go

```diff
@@ -196,9 +196,7 @@ func (b *dockerBuilder) expandDockerfile(templatesDir string, data map[string]in
 
 func (b *dockerBuilder) dockerBuild(variant string) (string, error) {
```

Comment:
`variant` not used anymore here, please remove the parameter from these methods.

Reply:
I tried that, but then the loop for the existing variants does not make sense. As I'm not a domain expert, not sure if it's connected to the Variants attribute in the types struct, which is used at the Dockerfile template for the agent, and finally connected to the `variants` array in the YAML file... 😱

Reply:
Well, you can leave in other places by now, removing it probably requires further refactors, but at least in this method it is not used anymore.
```suggestion
func (b *dockerBuilder) dockerBuild() (string, error) {
```